src/macros.rs

```diff
@@ -184,9 +184,6 @@ macro_rules! impl_hookable {
     impl_hookable!(@impl_pair ($($nm : $ty),*) (extern "win64"    fn($($ty),*) -> Ret));
     impl_hookable!(@impl_pair ($($nm : $ty),*) (extern "C"        fn($($ty),*) -> Ret));
     impl_hookable!(@impl_pair ($($nm : $ty),*) (extern "system"   fn($($ty),*) -> Ret));
-
-    #[cfg(feature = "thiscall-abi")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "thiscall-abi")))]
```

Comment:
Would you be able to keep this? Even though this is stabilized, it would force users to upgrade to newest stable and I want this to continue to work on older versions.